ecosystem/typescript/sdk/src/api/data-contracts.ts

```diff
@@ -685,6 +702,14 @@ export interface WriteModule {
   /** @example write_module */
   type: string;
 
+  /**
+   * All bytes data are represented as hex-encoded string prefixed with `0x` and fulfilled with
+   * two hex digits per byte.
+   *
+   * Different with `Address` type, hex-encoded bytes should not trim any zeros.
+   */
```

Comment:
this reads weird, I think you simply want to say that Unlike `Address`, leading bytes are not trimmed.

Same as below and above and ...

Reply:
This is directly from the openapi yaml haha